src/applications/proxy-rewrite/sass/style-consolidated.scss

```diff
@@ -106,58 +40,2951 @@ $font-stack: "Source Sans Pro", "Helvetica Neue", "Helvetica", "Roboto", "Arial"
   unicode-range: U+0000-00FF, U+0131, U+0152-0153, U+02C6, U+02DA, U+02DC, U+2000-206F, U+2074, U+20AC, U+2212, U+2215;
 }
 
-.consolidated {
-  font-family: $font-stack;
-  @import '~@department-of-veterans-affairs/component-library/dist/main.css';
+.ts-header-container,
```

Comment:
This file is intended to house all of the styles that Teamsites header/footer need to render. There **might** still be things coming from DST packages elsewhere, but my intention was to get rid of all of those connections.